packages/data-provider/src/services/submissionService.ts

```diff
@@ -321,6 +321,7 @@ const service = (dependencies: BaseDependencies) => {
 						data: record,
 						dictionaryCategoryId: categoryId,
 						entityName,
+						isValid: false, // default as false
```

Comment:
this could use a comment explaining where it gets redefined as true

Reply:
added comment in the code
`By default New Submitted Data is created as invalid until validation process proves otherwise`